README.md

```diff
@@ -25,26 +25,29 @@ See https://github.com/fossology/fossology/releases for source code download of
 
 For installation instructions see [Github Wiki](https://github.com/fossology/fossology/wiki)
 
-## Docker
-
-FOSSology comes with a Dockerfile allowing the containerized execution
-both as single instance or in combination with an external PostgreSQL database.
+### Install as a single container with all services combined
 **Note:** It is strongly recommended to use an external database for production
```

Comment:
You also need to preserve the repository, without that you only have the metadata but not the files!

Reply:
I'm not sure what you mean here. Could you explain?

Reply:
The data of the FOSSology application is stored in two places:
- the postgres DB, that contains the relational data and
- the repository at `/srv/fossology/repository/` containing the files, generated reports and other stuff

So for production you need to externalize the DB and the repostory.

Reply:
But the point is that the single instance Docker install doesn't provide an *external* db connection. Thus we get a warning and this is why we should likely deprecate it. I feel the installation instructions using this method should be removed as they are not production ready.

Reply:
There are several ways of getting it production ready:
- running a DB on the host, mounting the repository to the host and starting it with `--net=host`
- overwriting the DB connection via environment variables and putting the repository into a volume

Reply:
The single docker instance provides the possibility to configure an external db connection via env variables

Reply:
Visiting this a year later and it is unclear what you're after @maxhbr - if you're interested in the description of how one configures and database that is on a separate host (i.e. not _in_ the container nor on the host), then perhaps that is a separate set of documentation?